src/promptflow-devkit/promptflow/_dependencies/_pf_evals.py

```diff
@@ -0,0 +1,9 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# This file captures promptflow-evals dependencies on private API of promptflow.
+# In case changes are made please reach out to promptflow-evals team to update the dependencies.
+
+from promptflow._sdk._constants import LINE_NUMBER, Local2Cloud
```

Comment:
turns out this will raise flake8 error

Reply:
Shall I go ahead and ignore the flake8 error